src/openssl/mod.rs

```diff
@@ -0,0 +1,987 @@
+//! Module that uses the OpenSSL library to offer Elliptic Curve Verifiable Random Function (VRF) funcionality.
+//! This module follows the algorithms described in [VRF-draft-04](https://tools.ietf.org/pdf/draft-irtf-cfrg-vrf-04) and [RFC6979](https://tools.ietf.org/html/rfc6979).
+//!
+//! In particular, it computes:
+//!
+//! * The ECVRF_hash_to_curve uses the ECVRF_hash_to_curve_try_and_increment algorithm in [VRF-draft-04](https://tools.ietf.org/pdf/draft-irtf-cfrg-vrf-04)
+//! * The ECVRF_nonce_generation is specified in Section 3.2 of [RFC6979](https://tools.ietf.org/html/rfc6979)
+//!
+//! Note that in our case the input data is public, and thus possible information leaks in the form of timing side channels are irrelevant.
```

Comment:
Our case? Being this a public crate, we may prefer to put here a warning or disclaimer explaining that those attacks are possible on private data inputs.